packages/Calendar/src/index.js

```diff
@@ -16,9 +18,11 @@ const styles = StyleSheet.create({
 });
 
 function getMonthCount({ width }, value) {
-  const date = Array.isArray(value) ? value[0] : value;
-  return width > 400
-    ? [date, new Date(date.getFullYear(), date.getMonth() + 1)] : [date];
+  if (Array.isArray(value)) {
```

Comment:
Isn't this always true ?

Reply:
no , at first the value is a simple date object